app/views/layouts/application.html.erb

```diff
@@ -31,6 +31,8 @@
 
     <main role="main" id="content" class="<%= @content_item.schema_name.dasherize %>" lang="<%= I18n.locale %>">
       <span id="Top"></span>
+      <%= render partial: "govuk_web_banners/recruitment_banner" if recruitment_banner.present? %>
```

Comment:
Ideally this would be `<%= render "govuk_web_banners/recruitment_banner" %>`

Reply:
Thanks - yeah, I've done that.